cmd/web-handlers.go

```diff
@@ -754,8 +753,13 @@ next:
 			if _, err := globalBucketMetadataSys.GetLifecycleConfig(args.BucketName); err == nil {
 				hasLifecycleConfig = true
 			}
+			os := newObjSweeper(args.BucketName, objectName)
+			opts = os.GetOpts()
```

Comment:
if version is suspended, overwriting the empty version with "null" would cause a permanent delete of that version as opposed to setting a delete marker - same bug in all the other delete calls